ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CaseEvent.java

```diff
@@ -35,11 +35,11 @@
     REFER_TO_JUDGE_BY_CLAIMANT("ReferToJudgeByClaimant"),
     REFER_TO_JUDGE_BY_DEFENDANT("ReferToJudgeByDefendant"),
     SEALED_CLAIM_UPLOAD("SealedClaimUpload"),
+    DEFENDANT_PIN_LETTER_UPLOAD("DefendantPinLetterUpload"),
```

Comment:
Chris was saying we shouldn't upload Pin letter document due to ITHC, so please remove

Reply:
I am aware of this. I have deliberately not removed this because it will break changes done for ROC-4878. I will remove this as part of the story to remove pin document upload. I hope that is ok.

Reply:
Ah yes would be good not to interfere with John's story at this stage. @dharmendrak I will write a story to remove the PIN LETTER stuff, it can be done as part of that 👍

Reply:
👍

Reply:
Here is the story for reference: [ROC-5610] Never store PIN Letter documents